src/java.base/share/classes/jdk/internal/logger/LoggerFinderLoader.java

```diff
@@ -122,24 +107,13 @@ private static ErrorPolicy configurationErrorPolicy() {
     // Whether multiple provider should be considered as an error.
     // This is further submitted to the configuration error policy.
     private static boolean ensureSingletonProvider() {
-        return GetBooleanAction.privilegedGetProperty
-            ("jdk.logger.finder.singleton");
+        return Boolean.getBoolean("jdk.logger.finder.singleton");
     }
 
-    @SuppressWarnings("removal")
     private static Iterator<System.LoggerFinder> findLoggerFinderProviders() {
-        final Iterator<System.LoggerFinder> iterator;
-        if (System.getSecurityManager() == null) {
-            iterator = ServiceLoader.load(System.LoggerFinder.class,
+        final Iterator<System.LoggerFinder> iterator =
```

Comment:
Could be simply be `return ` ... no need for a local var.

Reply:
Done